pymc3_hmm/utils.py

```diff
@@ -172,15 +177,19 @@ def tt_broadcast_arrays(*args):
     """
     p = max(a.ndim for a in args)
 
-    args = [tt.shape_padleft(a, n_ones=p - a.ndim) if a.ndim < p else a for a in args]
+    list_args = [
+        tt.shape_padleft(a, n_ones=p - a.ndim) if a.ndim < p else a for a in args
+    ]
```

Comment:
Instead of renaming this variable, I think we can make `args` a tuple from the start (e.g. `args = tuple(tt.shape_padleft...)`).